src/devices/Amg88xx/Amg88xxUtils.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using UnitsNet;
+
+namespace Iot.Device.Amg88xx
+{
+    /// <summary>
+    /// This class contains utilities for working with AMG88xx devices.
+    /// </summary>
+    public static class Amg88xxUtils
+    {
+        /// <summary>
+        /// Temperature resolution of a pixel (in degrees celsius)
+        /// </summary>
+        public const double PixelTemperatureResolution = 0.25;
```

Comment:
Then if it's a public Temperature, you should use UnitsNet. If it's private, then double is fine